Resources/Prototypes/Alerts/alerts.yml

```
@@ -625,3 +619,19 @@
     state: deflecting0
   name: alerts-deflecting-name
   description: alerts-deflecting-desc
+
+# WD EDIT START
+- type: alert
+  id: KnockedDown
+  icons: 
```

yamllint

[error] trailing spaces

(trailing-spaces)